### PR TITLE
Create CharacterSet from a parsed unicode range string

### PR DESCRIPTION
**Pull request by angelcabo**

### CODE REVIEW

File: lib/characterset.js [hunk 1]
```diff
@@ -449,5 +449,41 @@
     return this.toArray().map(this.encodeCodePoint.bind(this)).join('');
   };
 
+  /**
+   * @param {string} input
+   */
+  CharacterSet.prototype.parseUnicodeRange = function (input) {
```
Comment (bramstein) [inline lib/characterset.js]: I was thinking about moving this to a static function on the CharacterSet class, for example:

```
CharacterSet.parseUnicodeRange = function (input) {
  var result = new CharacterSet();
  // parse and add code points to "result"

  return result;
};
```

**Comment by bramstein** [comment]
Also, if you don't mind, an addition to documentation in the README would be awesome.

File: lib/characterset.js (hunk 1 above)
Comment (angelcabo) [inline lib/characterset.js]: great idea. Should have kept it like that. Thanks

**Comment by bramstein** [comment]
👍

**Pull request merged**